typedoc.json

```diff
@@ -3,6 +3,20 @@
         "packages/*"
     ],
     "exclude": [
+        "packages/cjs-module-analyzer",
+        "packages/compartment-mapper",
+        "packages/daemon",
+        "packages/env-options",
```

Comment:
um

Reply:
My impression is that env-options is an implementation detail. It's useful for _building_ endo, but not so much for _using_ it. Not so?

Reply:
The only issue is if we want to empower the user to develop with conditionals on environment variables they define. If so, we should expose this. Certainly, we don't need to expose it immediately, as none of their use of anything else hinges on their ability to use the env-options API. But I think it is a nice development convenience. 

(Note: until recently I hated the notion because environment variables leak across boundaries in a security hostile manner. So if we do encourage, we should be clear that it is for development-time uses, like switching instrumentation on or off, or changing verbosity level, or...)

Reply:
I took env-options out of the exclude list. It's not really hurting anything.